ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -105,6 +103,15 @@ queue ${ARTIFACT_DIR}/oc_cmds/subscriptions oc --insecure-skip-tls-verify --requ
 queue ${ARTIFACT_DIR}/clusterserviceversions.json oc --insecure-skip-tls-verify --request-timeout=5s get clusterserviceversions --all-namespaces -o json
 queue ${ARTIFACT_DIR}/oc_cmds/clusterserviceversions oc --insecure-skip-tls-verify --request-timeout=5s get clusterserviceversions --all-namespaces
 queue ${ARTIFACT_DIR}/releaseinfo.json oc --insecure-skip-tls-verify --request-timeout=5s adm release info -o json
+queue ${ARTIFACT_DIR}/clusterroles.json oc --insecure-skip-tls-verify --request-timeout=5s get clusterroles --all-namespaces -o json
+queue ${ARTIFACT_DIR}/oc_cmds/clusterroles_rbac oc --insecure-skip-tls-verify --request-timeout=5s describe clusterroles.rbac
+queue ${ARTIFACT_DIR}/oc_cmds/clusterroles oc --insecure-skip-tls-verify --request-timeout=5s get clusterroles --all-namespaces
```

Comment:
We already collect json of clusterroles, why would we want a list of them additionally?

Reply:
Hey @vrutkovs , sorry for my ignorance: I tend to use the oc_cmds folder to look for objects, in this case clusterroles. As per your comment, I understand we might have that object available elsewhere. 

Could you point us where we might fetch it ? Thanks.

Reply:
What I'm saying is we already collect all the information about clusterroles in JSON format (`get clusterroles --all-namespaces -o json`), why do we also need a list of clusterrole names?

Reply:
ok thanks, I get it now, thought you were indicating this object was actually fetched prior to this PR. I guess we can remove the json part, what is your take @maneeshmehra ?